alerts.php

```diff
@@ -340,8 +340,13 @@ function ExtTransports($obj) {
             $msg        = FormatAlertTpl($obj);
             $obj['msg'] = $msg;
             echo $transport.' => ';
-            eval('$tmp = function($obj,$opts) { global $config; '.file_get_contents($config['install_dir'].'/includes/alerts/transport.'.$transport.'.php').' return false; };');
-            $tmp = $tmp($obj,$opts);
+            if ($transport == "canopsis") {
```

Comment:
I dont like this part, any reason why you dont use fully qualified class names instead?